src/test/compile-fail/privacy-sanity.rs

```diff
@@ -81,18 +81,18 @@ fn main() {
     }
     struct Ts(pub u8);
 
-    pub impl MarkerTr for .. {} //~ ERROR unnecessary visibility qualifier
-    pub impl Tr for S {  //~ ERROR unnecessary visibility qualifier
-        pub fn f() {} //~ ERROR unnecessary visibility qualifier
-        pub const C: u8 = 0; //~ ERROR unnecessary visibility qualifier
-        pub type T = u8; //~ ERROR unnecessary visibility qualifier
+    pub impl MarkerTr for .. {} //~ ERROR invalid visibility qualifier
+    pub impl Tr for S {  //~ ERROR invalid visibility qualifier
+        pub fn f() {} //~ ERROR invalid visibility qualifier
+        pub const C: u8 = 0; //~ ERROR invalid visibility qualifier
+        pub type T = u8; //~ ERROR invalid visibility qualifier
```

Comment:
This change seems to be for the worse, since `pub` is a valid qualifier, just unneccessary here